presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1128,6 +1128,33 @@ public void testCreateTable()
                         true,
                         ImmutableList.of(),
                         Optional.of("test")));
+
+        assertStatement("CREATE TABLE foo (a VARCHAR, b BIGINT COMMENT 'hello world', c IPADDRESS)",
+                new CreateTable(QualifiedName.of("foo"),
+                        ImmutableList.of(
+                                new ColumnDefinition(identifier("a"), "VARCHAR", emptyList(), Optional.empty()),
+                                new ColumnDefinition(identifier("b"), "BIGINT", emptyList(), Optional.of("hello world")),
+                                new ColumnDefinition(identifier("c"), "IPADDRESS", emptyList(), Optional.empty())),
+                        false,
+                        ImmutableList.of(),
+                        Optional.empty()));
+
+        assertEquals(
+                formatSql(
+                        new CreateTable(QualifiedName.of("foo"),
+                                ImmutableList.of(
+                                        new ColumnDefinition(identifier("values"), "BIGINT", emptyList(), Optional.empty()),
+                                        new ColumnDefinition(identifier("ORDER"), "VARCHAR", emptyList(), Optional.empty()),
+                                        new ColumnDefinition(quotedIdentifier("group"), "VARCHAR", emptyList(), Optional.empty())),
+                                false,
+                                ImmutableList.of(),
+                                Optional.empty()),
+                        Optional.empty()),
+                "CREATE TABLE foo (\n" +
+                        "   \"values\" BIGINT,\n" +
+                        "   \"ORDER\" VARCHAR,\n" +
+                        "   \"group\" VARCHAR\n" +
```

Comment:
Can you have a column that isn't a reserved word, so that we can see that it didn't get quoted.

Reply:
Done